en/getting_started/download_and_install.md

```diff
@@ -56,15 +56,20 @@ To install *QGroundControl* for Ubuntu Linux 16.04 LTS or later:
    ./QGroundControl.AppImage  (or double click)
    ```
 
-## Android
+## Android {#android}
 
 Install *QGroundControl* for Android 5.1 or later:
 
-1. Open the Google Play Store [QGroundControl link](https://play.google.com/store/apps/details?id=org.mavlink.qgroundcontrol).
-1. Follow the installation instructions.
+1. Download and install the stable build: [QGroundControl.apk](https://qgroundcontrol.s3-us-west-2.amazonaws.com/latest/QGroundControl.apk)
+   - Android phones have different ways of locating downloaded files.
```

Comment:
FYI only, too many possible ways to explain. This at least encourages people to understand they need to look for a file manager or whatever download system is used on their device.